### ISSUE TITLE
Offline maps broken in current play store version?

### ISSUE THREAD

**Issue by kk7ds** [ge0rg/aprsdroid#328]
I've been unable to get offline maps working with the current play store version. It seems like there might be some hint of similar troubles in #243 as well. Reproduced on a Pixel 3 and Pixel 5 running Android 12.

With the gl-2018-11-18_osm built everything works fine, using maps either from mapsforge.org's v3 directory, or the openandromaps "v3" download files. However, if I install the latest play store version, I get one of two behaviors. With the mapsforge files, as soon as I select the map file, aprsdroid crashes. With the openandromaps files, I can select the file, but when I display the map view in aprsdroid, I get a "Error loading $filename" bubble message at the bottom.

This is very annoying because the play store wants to constantly update (and thus break) aprsdroid, so I have to keep updates set to manual and select everything but aprsdroid every time I want to get updates.

Is there anything we can do to help track this issue down so it can be fixed? Thanks!

**Comment by ge0rg** [comment]
Yes, the current Play store release doesn't properly support offline maps. The old version of MapsForge I'm using isn't compatible with Android 12, and I can't publish a working version of the app to Google Play without some significant work. Please try the version at https://aprsdroid.org/download/builds/aprsdroid-gl-2022-02-07.apk which should support both Google and OSM maps.

**Comment by kk7ds** [comment]
Okay, the wiki here says it works with the current play store version. I spent a lot of time trying to make it work, on various devices and with lots of different maps. Can I propose an edit to change that for clarity?

**Comment by iotool** [comment]
I think you need download a map AND theme.

**Comment by na7q** [comment]
This is now fixed with this PR.
https://github.com/ge0rg/aprsdroid/pull/405
